modules/travel_pay/app/services/travel_pay/token_service.rb

```diff
@@ -6,16 +6,50 @@ class TokenService
     # returns a hash containing the veis_token & btsss_token
     #
     def get_tokens(current_user)
-      veis_token = token_client.request_veis_token
-      btsss_token = token_client.request_btsss_token(veis_token, current_user)
-
-      { veis_token:, btsss_token: }
+      cached = cached_by_account_uuid(current_user.account_uuid)
+      if cached
+        Rails.logger.info('BTSSS tokens retrieved from cache',
+                          { request_id: RequestStore.store['request_id'] })
+        { veis_token: cached.veis_token, btsss_token: cached.btsss_token }
+      else
```

Comment:
Do you think we need a case for: tokens cached, under 60 mins, but expired? I'm thinking in case the TTL is shortened on the API side in the future, or if we run into weird timing issues.

Reply:
I was actually thinking about this - I was thinking we should maybe shorten the TTL to just under the expiration of the tokens, so like 3300 instead of 3600 (shave off 5 minutes) just to give us a little bit of a buffer for if someone tries to use the cached tokens at the 3600 mark and they've _just_ expired.

Reply:
Done.